identity/src/main/java/com/android/identity/PresentationHelper.java

```diff
@@ -998,6 +1010,15 @@ public void setListener(@Nullable Listener listener, @Nullable Executor executor
         mDeviceRequestListenerExecutor = executor;
     }
 
+    /**
+     * Sets the object used to send back APDUs to the remote NFC device.
+     *
+     * @param nfcResponder a {@link PresentationHelper.NfcResponder} instance.
+     */
+    public void setNfcResponder(@NonNull NfcResponder nfcResponder) {
```

Comment:
This is kind of tricky to use correctly.

It seems super odd that we'd ever allow a `PresentationHelper` to be created, then an NFC session started, without setting this initially. I could even imagine someone never setting this, only testing with QR, then finding out that things break when they encounter and NFC device.

IMO this should be a constructor parameter, and if it's not passed there then NFC is disabled.